app/components/weather/WeatherCell.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import styles from './WeatherCell.scss';
+import windImage from '../../static/wind.compile_large.png';
+
+
+function parseWind(wind) {
+    const words = wind.split(' ');
+    if(words.length == 4) {
+        return (parseInt(words[0]) + parseInt(words[2])) / 2;
+    }
+    else
+        return parseInt(words[0]);
+}
+
+function parsePrecip(forecastDetails) {
+    const details = forecastDetails.toLowerCase();
+    if(details.match(/(little or no .+ accumulation expected)/g)) {
+        return .01;
+    }
+    const words = details.split('accumulation of');
+    if (details.match(/( .+ accumulation of)/g) || details.includes('inch')) {
+        if (details.includes('less than')) {
+            if (details.includes('tenth of an inch') || details.includes('half inch'))
+                return '<.1';
+            if (details.includes('half an inch') || details.includes('half inch'))
+                return '<.5';
+            if (details.includes('one inch'))
+                return '<1';
+        }
+        else if (details.includes('inches')) {
+            const inches = details.match(/(\d+)( to )(\d+)( inches)/i);
+            return (parseInt(inches[1]) + parseInt(inches[3])) / 2;
+        }
+        return words[1];
+    }
+    // not tested 
+    if (details.match(/(amounts between a.+ )(.+ )( and )(.+ )( of an inch)/g)) {
+        const inches = details.match(/(amounts between a.+ )(.+ )( and )(.+ )( of an inch)/i);
+        if (inches[4] == 'tenth')
+            return '<.1';
+        if (inches[4] == 'quarter')
+            return '<.25';
+        if (inches[4] == 'half')
+            return '<.5';
+        if (inches[4] == 'three quarters')
+            return '<.75';
```

Comment:
could use <½, ¼, ¾ because unicode is fun 😄

Reply:
also, there's repeated magic-number indexing of "`inches[4]`" - want to give that variable a name?

Reply:
ah good call!

Reply:
It's definitely a pain to parse all of this out, it would be so much nicer to have the raw data! Like the wind speed that are actually provided!